source/gameshared/gs_pmove.c

```diff
@@ -19,12 +19,17 @@ Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
 */
 
 #include "q_arch.h"
+#include "q_cvar.h"
```

Comment:
this hack works for the integrated server but it will crash in a real game. i think you need to use configstrings so it actually sends it over the network..